src/middlewares/json-api-vercel.ts

```diff
@@ -0,0 +1,45 @@
+import { Operation, VercelRequest, VercelResponse } from "../types";
+
+import Application from "../application";
+import ApplicationInstance from "../application-instance";
+
+import {
+  authenticate,
+  urlData,
+  handleBulkEndpoint,
+  handleJsonApiEndpoint,
+  convertErrorToHttpResponse
+} from "../utils/http-utils";
+
+export default function jsonApiVercel(app: Application) {
+  return async (req: VercelRequest, res: VercelResponse) => {
+    const appInstance = new ApplicationInstance(app);
+
+    try {
+      await authenticate(appInstance, req);
+    } catch (error) {
+      res.status(error.status);
+      res.json(convertErrorToHttpResponse(error));
+    }
+
+    if (process.env.AWS_REGION) {
```

Comment:
This should IMO be added into the readme. This is a behavioural change that depends on the context of the app and could catch more than one dev unaware.

Reply:
I ended up removing this, it's unnecessary.